### ISSUE TITLE
gen-macos.sh is complaining "Boost not found"

### ISSUE THREAD

**Issue by gui0506** [Vita3K/Vita3K#677]
I am using mac os 10.15.4 Catalina and when I try to build the code with "gen-macos.sh", I am getting the following errors:
```
++ cmake --find-package -DNAME=Boost -DCOMPILER_ID=GNU -DLANGUAGE=C -DMODE=EXIST
CMake Error: Problem processing arguments. Aborting.

+ BOOST_FOUND='Boost not found.'
```

**Comment by 1whatleytay** [comment]
Oh boy...

Have you installed boost over homebrew? `brew install boost`

If you still get the error after installing boost that would be a problem. But you could be able to work around it by just using cmake:
```
mkdir build
cd build
cmake ..
```

**Comment by gui0506** [comment]
ah, yes... that was the issue. I did not know that was a library. Thanks so much! 

As a follow-up question, the README says 

> When prompted to create schemes, create one for the emulator target only. The project builds many targets, so it will make your life easier if you create schemes as needed.

Seems like there is no such target. Is the doc outdated or am I missing anything? thanks

**Comment by 1whatleytay** [comment]
Docs' outdated. It's called vita3k now. Safe to close?

**Comment by gui0506** [comment]
The build is failing for some reason but I guess that's a separate issue. I would like to join the discord server for developer help.  Can you invite me?

**Comment by 1whatleytay** [comment]
Here :) https://discordapp.com/invite/6aGwQzh

**Comment by ichisadashioko** [comment]
This might be fixed in PR #686
